NitroxServer-Subnautica/Program.cs

```diff
@@ -110,20 +120,138 @@ private static async Task StartServer(string[] args)
             AppMutex.Release();
         }
 
-        await listenForCommands;
+        await handleConsoleInput;
+
+        Console.Write($"{Environment.NewLine}Server is closing..");
```

Comment:
In external mode, we cannot see this text appearing

Reply:
It's there now after the errors were fixed. But in embedded, it closes faster than external so you won't be able to read it.